.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: node_js
 node_js:
-  - "0.12"
+  - "4.0"
 before_install:
   - rvm install 2.2.2
```

Comment:
@ncoden none that I'm aware of but I am not 100% sure that scss-lint is the last thing Ruby related in the repo so I wanted to verify that before removing rvm in the travis scripts.